marconi-chain-index/src/Marconi/ChainIndex/Experimental/Run.hs

```diff
@@ -25,25 +30,43 @@ run appName = do
       includeScript = True
       socketPath = Cli.optionsSocketPath $ Cli.commonOptions o
       networkId = Cli.optionsNetworkId $ Cli.commonOptions o
+      retryConfig = Cli.optionsRetryConfig $ Cli.commonOptions o
+      preferedStartingPoint = Cli.optionsChainPoint $ Cli.commonOptions o
```

Comment:
Same here, especially as, as we discussed, we don't have a clear semantic for this option. Unless there's a real need to add it now, I would postpone it.

Reply:
For this one, it was pretty important. The reason is that we needed a way to start syncing from the tip of the local node, so that we can test what happens when rollbacks occur on mainnet. It was also trivial to implement because it's the same logic as the old indexers. 

Of course, the semantics would be eventually enhanced as we discussed, but the current implementation is necessary for testing purposes.